src/page/login.js

```diff
@@ -0,0 +1,45 @@
+import { Form,Button } from "../components/index";
+import {useState} from "react"
+import {useNavigate} from "react-router-dom"
+import  Profile  from "./Profile";
+const Login =()=>{
+
+    const [users,setUsers] =useState(
+        {
+            name: "",
+            email:"",
+            Password:"",
+        }
+    )
+    let navigate = useNavigate();
+    const HandleChange =() =>{
+      var data=  setUsers({
+            ...users,
+            name:document.getElementById("name").value,
```

Comment:
You can directly retrieve value by taking event.target.value and pass it to object

Reply:
Ok i'll update this